README.md

```diff
@@ -49,16 +49,16 @@ Where `llvm_build_dir` is the LLVM build directory.
 
 ### LLVM in-tree build
 
-The translator can be built as a regular LLVM subproject. To do that you need to clone it to `llvm/projects` or `llvm/tools` directory. 
+The translator can be built as a regular LLVM subproject. To do that you need to clone it into the `llvm/projects` or `llvm/tools` directory.
 ```
-git clone http://llvm.org/git/llvm.git
-cd llvm/projects
+git clone https://github.com/llvm/llvm-project.git
```

Comment:
Does it still work? (I haven't tried)

For example, for intel/llvm, the way we build the translator within monorepo is to pass a few additional cmake arguments, something like:

```
-DLLVM_EXTERNAL_PROJECTS="llvm-spirv" \
-DLLVM_ENABLE_PROJECTS="llvm-spirv" \
-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=llvm/llvm-spirv \
```

Reply:
That's an alternative way of doing it.  Putting it directly into `llvm/projects` avoids the need to pass those extra cmake flags.

I checked that the instructions are still working before opening this PR.

Reply:
Thanks!